code/modules/projectiles/projectile/special/saboteur.dm

```diff
@@ -1,3 +1,7 @@
+///Override on subtype to add behaviour. Whatever happens when we are sabotaged
+/datum/proc/on_saboteur(datum/source, disrupt_duration)
```

Comment:
Make this an atom proc and keep the signal, otherwise it won't work for the seclight_attachable component.

Reply:
I've made the proc back into an atom proc but I made it a parent call where it sends a signal so the 2 datums can register to it for ethereals and attached seclites